cmd/boost/direct_deal.go

```diff
@@ -38,8 +38,9 @@ var directDealAllocate = &cli.Command{
 		},
 		&cli.StringSliceFlag{
 			Name:     "pieceInfo",
-			Usage:    "data pieceInfo[s] to create the allocation. The format must be pieceCid1=pieceSize1 pieceCid2=pieceSize2",
+			Usage:    "data pieceInfo[s] to create the allocation. The format must be --pieceInfo pieceCid1=pieceSize1 --pieceInfo pieceCid2=pieceSize2",
```

Comment:
In general for param names we use snake-case, let's do that for piece-info, term-min and term-max